cirq/ops/gate_operation.py

```diff
@@ -15,7 +15,7 @@
 """Basic types defining qubits, gates, and operations."""
 
 from typing import (Any, Dict, FrozenSet, List, Optional, Sequence, Tuple, Type,
-                    TypeVar, Union, TYPE_CHECKING)
+                    TypeVar, Union, TYPE_CHECKING, Iterable)
```

Comment:
nit: keep alphabetical order

Reply:
Done. I think we should either write a tool to do this automatically or stop caring about it.